src/small-component.scss

```diff
@@ -15,3 +151,17 @@
 	word-break: break-all;
 	word-wrap: break-word;
 }
+
+.wb-fieldflow-form .btn {
```

Comment:
Is those need to be specific to fieldflow? Can they be incorporated with the ```.gc-form``` style?

Reply:
This could be an alternative location if we can exclude fieldflow. https://github.com/wet-boew/GCWeb/blob/master/src/_defaults.scss#L225

Reply:
Unfortunately the button ends up falling outside of the .gc-form wrapping div - this seemed the best way to target it

Reply:
Only the CSS property/instruction of each section that need to be in alphabetical order, not the CSS selector that represent a section. Sorry for the confusion

Reply:
An alternative would be to create a SASS file for fieldflow, so that SASS file woudl be located beside the fieldflow plugin.